projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -20,6 +20,22 @@ const showError = async (error: any) => {
   );
 };
 
+interface IJobConvertToNotebookMessage {
+  code_structure: string[];
+  removed_files: string[];
+}
+
+type jobConvertToNotebookRequestResult = {
```

Comment:
This is probably a little subjective so you don't need to make this change. 

But I would probably make two types. 

jobConvertToNotebookSuccess {
message: IJobConvertToNotebookMessage
}
jobConvertToNotebookFailure{
message: string
}

Reply:
I can introduce the 2 types you proposed, yeah, sounds good.

Reply:
Actually, then we would have to go and refactor all the empty string messages because it would not recognise "string" as their type when replaced by the 2 new types. I will not add them for now.